.actions/assistant.py

```diff
@@ -43,6 +43,10 @@
         "requirements/fabric/base.txt",
         "requirements/fabric/strategies.txt",
     ),
+    "data": (
+        "requirements/data/extra.txt",
```

Comment:
```suggestion
        "requirements/data/base.txt",
        "requirements/data/extra.txt",
```

Reply:
@borda we can't have this as base as it won't be optional this way. These dependencies are meant to be optional and only installed with `lightning[data]`, which is why we don't have a base here. Otherwise we'll have a lot of dependency conflicts with oldest versions